sdk/src/types/block/mana/allotment.rs

```diff
@@ -21,6 +21,18 @@ pub struct Allotment {
     pub(crate) mana: u64,
 }
 
+impl PartialOrd for Allotment {
+    fn partial_cmp(&self, other: &Self) -> Option<core::cmp::Ordering> {
+        Some(self.cmp(other))
```

Comment:
FOr `PartialOrd` you compare a whole`Allotment` but for `Ord` `account_id`. Is this correct?

Reply:
Partial just calls the Ord impl